components/02-elements/paragraph/_paragraph.scss

```diff
@@ -19,3 +19,17 @@ a {
     color: $color-main1;
     text-decoration: underline;
 }
+
```

Comment:
Wow... I missed this code somehow. Is shouldn't be added just like that in the global scope, b/c will affect the whole page.